components/announcement_bar/configuration_bar/configuration_bar.jsx

```diff
@@ -41,7 +41,7 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
     render() {
         // System administrators
         if (this.props.canViewSystemErrors) {
-            const renewalLink = RENEWAL_LINK + '?id=' + this.props.license.id + '&user_count=' + this.props.totalUsers;
+            const renewalLink = RENEWAL_LINK + '?id=' + this.props.license.Id + '&user_count=' + this.props.totalUsers;
```

Comment:
Could we change this to use back ticks instead of adding them together?
`${RENEWAL_LINKI'd........`. I think the proper name is template literals instead of backticks https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals .